From 41dcb2ba373ae8475944887c5d4558efc652e4fa Mon Sep 17 00:00:00 2001
From: Radu Pirea <radu-nicolae.pirea@nxp.com>
Date: Fri, 29 Jan 2021 13:03:00 +0200
Subject: [PATCH 69/76] s32: pinctrl: fix SPI pin configs

Enable output buffer for SPI SOUT pin.
Fix SPI5 mux ids.

Issue: ALB-6403

Signed-off-by: Radu Pirea <radu-nicolae.pirea@nxp.com>
---
 include/dt-bindings/pinctrl/s32-gen1-pinctrl.h | 2 +-
 include/dt-bindings/pinctrl/s32g274a-pinctrl.h | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/dt-bindings/pinctrl/s32-gen1-pinctrl.h b/include/dt-bindings/pinctrl/s32-gen1-pinctrl.h
index 46f5d5ef74..dd3cc097d3 100644
--- a/include/dt-bindings/pinctrl/s32-gen1-pinctrl.h
+++ b/include/dt-bindings/pinctrl/s32-gen1-pinctrl.h
@@ -43,7 +43,7 @@
 	SIUL2_MSCR_S32_G1_PUE | SIUL2_MSCR_S32_G1_PUS | \
 	SIUL2_MSCR_S32_G1_SRC_50MHz)
 
-#define DSPI_SOUT_PIN_CFG	(SIUL2_MSCR_S32_G1_IBE | \
+#define DSPI_SOUT_PIN_CFG	(SIUL2_MSCR_S32_G1_OBE | \
 	SIUL2_MSCR_S32_G1_SRC_50MHz)
 
 #define I2C_PIN_CFG		(SIUL2_MSCR_S32_G1_OBE | \
diff --git a/include/dt-bindings/pinctrl/s32g274a-pinctrl.h b/include/dt-bindings/pinctrl/s32g274a-pinctrl.h
index 5e8c9717fa..858d2cbb3d 100644
--- a/include/dt-bindings/pinctrl/s32g274a-pinctrl.h
+++ b/include/dt-bindings/pinctrl/s32g274a-pinctrl.h
@@ -85,12 +85,12 @@
 
 /* PA11 */
 #define PA11_MSCR_S32G2XX	(11)
-#define PA11_SPI5_SOUT_CFG	(SIUL2_MSCR_S32_G1_MUX_ID_1 | \
+#define PA11_SPI5_SOUT_CFG	(SIUL2_MSCR_S32_G1_MUX_ID_3 | \
 	DSPI_SOUT_PIN_CFG)
 
 /* PA12 */
 #define PA12_MSCR_S32G2XX	(12)
-#define PA12_SPI5_CS0_CFG	(SIUL2_MSCR_S32_G1_MUX_ID_1 | \
+#define PA12_SPI5_CS0_CFG	(SIUL2_MSCR_S32_G1_MUX_ID_3 | \
 	DSPI_CS_PIN_CFG)
 
 /* PA13 */
-- 
2.17.1

